matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/DeviceListManager.kt

```diff
@@ -377,7 +377,7 @@ internal class DeviceListManager @Inject constructor(private val cryptoStore: IM
         }
 
         // Update devices trust for these users
-        dispatchDeviceChange(downloadUsers)
+        // dispatchDeviceChange(downloadUsers)
```

Comment:
not necessary anymore?

Reply:
It was called twice, because it is called in onKeysDownloadSucceed (called 2 lines below this one)

Reply:
Yes indeed. I'm wondering if we should not keep this one and remove the other call, but OK anyway.